test/fff_test_c.c

```diff
@@ -6,14 +6,9 @@
 #define OVERRIDE_CALL_HIST_LEN 17u
 #define FFF_CALL_HISTORY_LEN OVERRIDE_CALL_HIST_LEN
 
-#include "../fff.h"
+#include "fff.h"
 #include "c_test_framework.h"
 
-#include <assert.h>
```

Comment:
Maybe I'm missing something here - but are these includes not needed?

Reply:
<assert.h> conflicts with how Zephyr does asserts. Instead of including it here directly, `test/c_test_framework.h` was moved to `test/framework/c_test_framework.h`. That file includes all these headers. In Zephyr's build we add our own framework headers to the path and define `ASSERT_EQ` and `ASSERT_TRUE` to map to the zephyr `zassert_equal` and `zassert_true`.